types/mapbox-gl/index.d.ts

```diff
@@ -771,6 +771,13 @@ declare namespace mapboxgl {
          */
         pitch?: number | undefined;
 
+        /**
+         * A style's projection property sets which projection a map is rendered in.
+         *
+         * @default 'mercator'
+         */
+         projection?: 'equalEarth' | 'albers' | 'mercator' | 'lambertConformalConic' | 'winkelTripel' | 'naturalEarth' | 'equirectangular';
```

Comment:
I don't really understand this project/library, but the link to documentation you posted (https://docs.mapbox.com/mapbox-gl-js/style-spec/projection/) has an example of projection that looks like this:

```json
"projection": {
  "name": "albers",
  "center": [
    -154,
    50
  ],
  "parallels": [
    55,
    65
  ]
}
```
so isn't the type of the `projection` property here supposed to be a structured type? Like `projection?: { name: 'equalEarth' | 'albers' | ...., parallels?: ..., center?: ... }`?